src/pages/bestStories/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import axios from "axios";
+import CONSTANTS from "../../constants/appConstant";
+import Story from "./../../components/Story/index";
+
+const { BASE_API_URL } = CONSTANTS;
+export default class TopStories extends Component {
+  state = {
+    topStories: [],
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    this.fetchTopStories();
+  }
+
+  fetchTopStories = async () => {
+    try {
+      const { data: topStoriesIds } = await axios.get(
+        `${BASE_API_URL}/beststories.json`
+      );
+      const stories = await Promise.all(
+        topStoriesIds.slice(0, 20).map((storyId) => this.getStory(storyId))
+      );
+      console.log(stories);
+      this.setState({ topStories: stories, isLoading: false });
+    } catch (e) {
+      console.log("error");
+    }
+  };
+
+  /**
+   *
```

Comment:
Use function description like this:
![image](https://user-images.githubusercontent.com/22271897/121469217-d1cfa800-c9db-11eb-9101-1f797f8a91b3.png)